Update information on deprecating exercises (#126)
exercism/docs: language-tracks/configuration/exercises.md

## Deprecating exercises

> Deleting an exercise is a destructive action as it would delete all users' solutions too. Instead, we allow exercises to be deprecated, which users who have completed the exercise can access their existing solutions, but those who have not will not see or be able to start that.

Reviewer: ```suggestion
Deleting an exercise is a destructive action as it would delete all users' solutions too. Instead, we allow exercises to be deprecated, which users who have completed the exercise can access their existing solutions, but those who have not will not see or be able to start.
```

> Deleting an exercise is a destructive action as it would delete all users' solutions too. Instead, we allowe exercises to be deprecated, which users who have completed the exercise can access their existing solutions, but those who have not will not see or be able to start that.

Reviewer 2: Typo?

> Instead, we allowe

Should read "allow".

> Deleting an exercise is a destructive action as it would delete all users' solutions too. Instead, we allowe exercises to be deprecated, which users who have completed the exercise can access their existing solutions, but those who have not will not see or be able to start that.

Reviewer 2: > Instead, we allowe exercises to be deprecated, which users who have completed the exercise can access their existing solutions, but those who have not will not see or be able to start that.

might be more clear with a small grammar change. Is this still accurate?

> Instead, we allow exercises to be deprecated. Users who have completed the exercise [...] but those who have not [...].

Deprecated exercises must have the following set in `config.json`:

- **uuid**
- **slug**
- **deprecated** (boolean, true)

> All other fields in deprecated exercises can be deleted, although this too is a destructive action as it removes information for legacy users, so we recommend keeping fields where possible.

Reviewer: It sounds like this should be considered as something NOT to do since it is a destructive action.

How about this instead:
```suggestion
All other fields in deperecated exercises should remain as is.
```
